markdown/epilogue.markdown

```diff
@@ -1,39 +1,167 @@
+imported into Python. By providing some type annotations to Python variables,
+the compiled C code can end up being a hundred or even a thousand times faster
+than comparable Python code. Cython is now an industry standard and is used in
+NumPy, SciPy, and many related libraries (such as scikit-image) to provide fast
+algorithms in array-based code. Kurt Smith has written the simply-titled
+[Cython](http://shop.oreilly.com/product/0636920033431.do) to teach you the
+fundamentals of this language.
+
+An often easier-to-use alternative to Cython is Numba, a just-in-time compiler
+(JIT) for array-based Python. JITs wait for a function to be executed once, at
+which point they can infer the types of all the function arguments and output,
+and compile the code into a highly efficient form for those specific types. In
+Numba code, you don't need to annotate types: Numba will infer them when a
+function is first called. Instead, you simply need to make sure that you only
+use basic types (integers, floats, etc), and arrays, rather than more
+complicated Python objects. In these cases, Numba can compile the Python code
+down to very efficient code and speed up computations by orders of magnitude.
+
+Numba is still very young, but it is already very useful. Importantly, it shows
+what is possible by Python JITs, which are set to become more commonplace: Python
+3.6 added features to make it easier to use new JITs (the Pyjion JIT is based on
+these). You can see some examples of Numba use, including how to combine it with
+SciPy, in Juan's blog at https://ilovesymposia.com/tag/numba/. And Numba,
+naturally, has its own very active and friendly mailing list.
+
+
+## Contributing to this book
+
+The source of this book is itself hosted on GitHub at
+https://github.com/elegant-scipy/elegant-scipy (also at http://elegant-scipy.org).
+Just as if you were contributing to any other open source project, you can raise
+issues or submit pull requests if you find bugs or typos — and we would very much
+appreciate it if you did!
+
+We used some of the best code we could find to illustrate the various parts of
+the SciPy and NumPy libraries. If you have a better example, please raise an issue
+in the repo! We would love to include it in future editions.
+
+We are also on Twitter, at [@elegantscipy](https://twitter.com/elegantscipy). Drop
+us a line if you want to chat about the book! The individual authors are
+[@jnuneziglesias](https://twitter.com/jnuneziglesias),
+[@stefanvdwalt](https://twitter.com/stefanvdwalt), and
+[@hdashnow](https://twitter.com/hdashnow).
+
+We particularly want to hear about it if you use any of the ideas or code in this
+book to advance your scientific research. That's the point of SciPy!
+
+In the meantime, we hope you enjoyed this book and found it useful. If so, tell
+all your friends, and come say hi on the mailing lists, at a conference, on
+GitHub, and on Twitter! Thanks for reading.
```

Comment:
[This](https://twitter.com/tomgara/status/860938163526799362) is priceless and surprisingly related:

> My friend's toddler babbled "don't forget to subscribe" as he was put to bed. Kid watches so much YouTube he thought it means "goodbye"

=D